include/iocore/eventsystem/EThread.h

```diff
@@ -691,6 +698,17 @@ EThread::Metrics::decay() -> self_type &
   return *this;
 }
 
+inline int
+EThread::get_numa_node()
+{
+  if (this->numa_node == -1) {
+    unsigned int cpu, node;
+    getcpu(&cpu, &node);
+    this->numa_node = node;
+  }
+  return this->numa_node;
```

Comment:
This should handle when numa.h is not available

Reply:
The getcpu() function is included in the numactl package. Adding a conditional directive should solve this issue.